CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ docker compose up
 Initializing the db can be done with:
 
 ```sh
-mix ecto.setup
+make core-setup
```

Comment:
The `cwd` is not `apps/core` and `ecto.setup` is not registered as an umbrella command.